app/components/blacklight/search_bar_component.rb

```diff
@@ -5,6 +5,7 @@ class SearchBarComponent < Blacklight::Component
     renders_one :append
     renders_one :prepend
     renders_one :search_button
+    renders_many :inputs
```

Comment:
I don't love this slot name; `before_input_groups` is kinda clunky, but might express the intent better?

Reply:
Yeah, I kind of punted on the naming. Being clear is a good idea 👍🏻